packages/react/src/Banner/Banner.examples.stories.tsx

```diff
@@ -0,0 +1,109 @@
+import {Banner} from '../Banner'
+import {action} from '@storybook/addon-actions'
+import Link from '../Link'
+import type {Meta} from '@storybook/react'
+import {Status} from '../internal/components/Status'
+import {Alert} from '../internal/components/Alert'
+import FormControl from '../FormControl'
+import RadioGroup from '../RadioGroup'
+import Radio from '../Radio'
+import {Button} from '../Button'
+import React from 'react'
+import {useFocus} from '../internal/hooks/useFocus'
+
+const meta = {
+  title: 'Drafts/Components/Banner/Examples',
+  component: Banner,
+} satisfies Meta<typeof Banner>
+
+export default meta
+
+export const WithUserAction = () => {
+  const [hasError, setHasError] = React.useState(false)
+  const bannerRef = React.useRef<React.ElementRef<typeof Banner>>(null)
+  const focus = useFocus()
+
+  return (
+    <>
+      {hasError ? (
+        <Banner
+          ref={bannerRef}
+          title="Error"
+          description={<Alert>Something went wrong. Please try again later.</Alert>}
```

Comment:
Though I understand that heading semantics won't be conveyed through a live region, I'm wondering whether it would make sense to encourage including it as part of the live region announcement since it may contain information that isn't part of the description. 🤔 

What would that look like?

Reply:
@khiga8 would something like this maybe work?

```tsx
description={
  <Alert>
    <VisuallyHidden>Error:</VisuallyHidden> Something went wrong. Please try again later.
  </Alert>
}
```

Reply:
To reduce redundancy around the heading text appearing in the banner twice for screen reader users, would something like this be possible?

```.tsx
<Banner>
  <Status>
    <Banner.Title>Subscription renewed</Banner.Title>
    <Banner.Description>Your subscription has been successfully renewed until May 5, 2024. </Banner.description>
  </Status>
</Banner>
```

cc: @ichelsea if you have other thoughts!

Reply:
@khiga8 I bet we could make it work, we could also announce the live region itself if that would be desirable (e.g. something like `Banner as={Status}`